packages/ag-charts-types/src/chart/cartesianOptions.ts

```diff
@@ -256,7 +272,7 @@ export interface AgOrdinalTimeAxisOptions<TContext = ContextDefault>
     /** Options for labels and ticks for the parent level intervals. */
     parentLevel?: AgTimeAxisParentLevel<TContext>;
     /** Configuration for the axis ticks interval. */
-    interval?: AgAxisContinuousIntervalOptions<AgTimeInterval | AgTimeIntervalUnit | number>;
+    interval?: AgAxisBandTimeIntervalOptions;
```

Comment:
For example, this has changed type, but it is the same result with just the `placement` option added.